appintro/src/main/java/com/github/appintro/AppIntro.kt

```diff
@@ -199,4 +200,22 @@ abstract class AppIntro : AppIntroBase() {
             bottomSeparator.visibility = View.INVISIBLE
         }
     }
+
+    /**
+     * Adds a margin to the bottom of the content of the slide equal to the bottom bar.
+     * For situations where the bar is not transparent, and you want the content of the slide to
+     * be intractable.
+     *
+     * @param setBarPadding Set : true for margin. false for no margin.
+     */
+    fun setBarPadding(setBarPadding: Boolean) {
+        val bottomBar = findViewById<View>(R.id.bottom)
+        val margin = if (setBarPadding)
+            resources.getDimension(R.dimen.appintro_bottombar_height).toInt()
+        else
+            0
+        (bottomBar.layoutParams as? ViewGroup.MarginLayoutParams)
+            ?.setMargins(0, 0, 0, margin)
+            ?.also { bottomBar.requestLayout() }
```

Comment:
I don't think this works. Specifically you should add marginBottom to `pager_gesture_overlay` and not to `bottom`. Could you add an example to the sample app to make sure this is not broken?

Reply:
True, it has now been fixed, and I've added an example.